content/assignments/minishell/checker/README.md

```diff
@@ -0,0 +1,31 @@
+# Testing and Grading
+
+The testing is automated.
+Tests are located in the `inputs/` directory.
+
+```console
+student@so:~/.../assignments/minishell/checker/_test/inputs$ ls -F
+test_01.txt  test_03.txt  test_05.txt  test_07.txt  test_09.txt  test_11.txt  test_13.txt  test_15.txt  test_17.txt
+test_02.txt  test_04.txt  test_06.txt  test_08.txt  test_10.txt  test_12.txt  test_14.txt  test_16.txt  test_18.txt
+```
+
+To execute tests you need to run:
+
+```bash
+./run_all.sh
```

Comment:
```suggestion
student@os:~/.../assignments/minishell/checker$ ./run_all.sh
```
And add a sample output that gets full marks.

Reply:
Done since README is no longer in this folder.
It's already in the PR with the assignment.
Add this review to this [PR] if you consider this useful.
[PR]: https://github.com/open-education-hub/operating-systems/pull/111